app/config/config.yml

```diff
@@ -29,6 +29,8 @@ framework:
 twig:
     debug:            "%kernel.debug%"
     strict_variables: "%kernel.debug%"
+    paths:
+        "%kernel.root_dir%/../custom/email_templates": custom_email_templates
```

Comment:
tried that. symfony will flat out refuse to bootstrap if that path does not exist.
so we'd have to ensure that the files dir, and it's subdir for custom templates exists post install.
perhaps a composer hook? 
thoughts?